setup-env/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'setup-env'
+description: 'Setup BrowserStack Test Environment'
+inputs:
+  username:
+    description: 'BrowserStack Username'
+    required: true
+  access-key:
+    description: 'BrowserStack Access Key'
+    required: true
+  build-name:
+    description: 'Build name for the tests'
+    required: false
+  project-name:
+    description: 'Project name for the tests'
+    required: false
+runs:
+  using: 'node12'
+  main: 'dist/index.js'
```

Comment:
Should we add cleanup jobs. If a customer is add running this a Github runner, he may want his sensitive information cleared from env variables.
Check with product, if this can be part of next releases.

Reply:
Valid case. But the environment is anyway destroyed post the completion of the job.
And as far as I have checked, there is no way to trigger a `cleanup-step` manually which will clean the environment variables set by us.
Or probably it can be a separate action altogether to clean up the environment just the way we set up the environment using `setup-env`.

Reply:
We don't need cleanup unless and until its a self-hosted runner where the env variables will persist. We will call that out in the documentation.

Reply:
Resolving this. Behaviour as expected by product.